doc/classes/MeshLibrary.xml

```diff
@@ -59,6 +59,13 @@
 				Returns the item's name.
 			</description>
 		</method>
+		<method name="get_item_navigation_layers" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="id" type="int" />
+			<description>
+				Returns the item's [code]navigation_layers[/code] bitmask.
```

Comment:
`[code]` markup with a snake_case identifier only makes sense if we're referring to actual code - but I don't see any `navigation_layers` property here, so this is confusing.

If this is an internal C++ member variable which is accessed via those methods, it doesn't make sense to refer to its C++ name which users don't know about. Instead, this should be described in plain English.

Reply:
Reading the code a bit more, I see it's a property defined dynamically for each Mesh item... not sure how best to describe this.

Reply:
I think just using plain text would work best here,
[code]navigation_layers[/code] -> navigation layers